docs/modules/ROOT/pages/path-to-production.adoc

```diff
@@ -0,0 +1,31 @@
+= Path to Production
+
+Aside from testing scaffolding and the Maven project structure, aiSSEMBLE(TM) generates several artifacts to help drive
+consistency, repeatability, and quality of delivery. These artifacts are designed as starting points for a mature
+DevOps-centric approach for delivering high-quality AI systems and are not intended as complete solutions in and of
+themselves.
+
+== Mapping to aiSSEMBLE Concepts
+[#img-you-are-here-path-to-production]
+.xref:solution-baseline-process.adoc[You Are Here]
+image::you-are-here-path-to-production.png[You Are Here,200,100,role="thumb right"]
+
+_Path to Production_: Many AI systems are developed across the industry much like a school project - a Data Engineer or
```

Comment:
A: instead of 'school project', feels like perhaps we can say 'leverage ad-doc, manual processes'

Reply:
✅ UPDATED